src/Gir/Marshal/IType.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace Gir
+{
+	public interface IType
```

Comment:
Is this complete yet? Is IType supposed to give information only about the native type or do we also add information on C# like CSharpType or IsBlittable here?

Reply:
We can make the blittable and co extensions. I'm trying to keep the interface simple, as in, what's the minimum set of things a marshallable type needs to have. It's not complete yet.